storage/gluster-health-check

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+GLUSTER_PODS=$(oc get pods -l glusterfs-node=pod --all-namespaces -o wide | tail -n +2 | awk '{print $2}')
+GLUSTER_POD_NAMESPACES=$(oc get pods -l glusterfs-node=pod --all-namespaces -o wide | tail -n +2 | awk '{print $1}')
+GLUSTER_POD_NODES=$(oc get pods -l glusterfs-node=pod --all-namespaces -o wide | tail -n +2 | awk '{print $8}')
+
+while read gluster_pod <&3 && read gluster_namespace <&4 && read gluster_node <&5; do
+  echo
+  echo
+  echo "GLUSTER STATUS FOR POD (${gluster_pod}) IN PROJECT (${gluster_namespace}) ON NODE (${gluster_node})"
+  oc project ${gluster_namespace}
```

Comment:
I'd suggest not changing the project but adding '-n ${gluster_namespace}' to the 'oc rsh' - stops this script interfering with other scripts/users and visa versa.

Reply:
done.

Reply:
actually, that dind't work

Reply:
okay, now it works.